src/brand/Makefile

```diff
@@ -103,11 +115,11 @@ clobber: clean
 	rm -f $(ROOTFILES)
 	rm -fr $(ROOTBRAND)
 
-install: $(ROOTFILES)
+install: $(ROOTFILES) $(BHYVEBRANDPKG)/createzone
```

Comment:
Why do you avoid adding $(BHYVEBRANDPKG)/createzone to ROOTFILES?

Reply:
My thinking was, that this should help not picking $(BHYVEBRANDPKG)/createzone from bhyve directory but kvm directory.

Reply:
No, it's not related, pick https://github.com/pyhalov/pkg5/commit/3f235072007c2f681b171429c1a8f00c4cac07d8

Reply:
Ah ok, done.